assets/src/application/services/toaster/ToasterPosition.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import getToasterPositionStyle from './utilities/getToasterPositionStyle';
+import Toast from './Toast';
+import { ToasterPositionProps } from './types';
+
+/**
+ * adapted from:
+ * @link https://github.com/bmcmahen/toasted-notes/blob/master/src/Alert/ToastManager.ts
```

Comment:
404, it's:


https://github.com/bmcmahen/toasted-notes/blob/master/src/Alert/ToastManager.tsx

Reply:
good catch